app/controllers/api/functions.php

```diff
@@ -1475,6 +1475,55 @@
         $response->noContent();
     });
 
+App::patch('/v1/functions/:functionId/deployments/:deploymentId/builds/:buildId')
```

Comment:
Not sure about the look of the REST API here, also the description doesn't follow our usual conventions. Also, only a delete request should return `no content`, should this be a delete request?

Should we maybe have a CRUD for `builds` and delete the build documents while updating the deployment status to cancelled?

@vermakhushboo @Meldiron @christyjacob4 would love your thoughts on the DX here.

Reply:
Updated the response model and description. `deployment` doesn't have a status. I don't think we should delete the build document, since the status of the build is displayed on the console. Open for suggestions.